docs/conf.py

```diff
@@ -68,14 +68,20 @@ def run(self):
 # ones.
 extensions = [
     'sphinx.ext.autodoc',
+    'sphinx.ext.autosummary',
     'sphinx.ext.todo',
     'sphinx.ext.viewcode',
     'sphinx.ext.napoleon',
 ]
 
+autodoc_default_flags = ['members', 'inherited-members', 'show-inheritance']
```

Comment:
@hayesall Thank you very much for quick fixes! Seems that this config value is deprecated. I think we can duplicate these setting in new `autodoc_default_options` config value to support all versions of Sphinx.
> Deprecated since version 1.8: Integrated into autodoc_default_options.
    https://www.sphinx-doc.org/en/2.0/usage/extensions/autodoc.html#confval-autodoc_default_flags

Reply:
@StrikerRUS Really glad you caught this. It doesn't look like sphinx raises a warning yet, but I know a few other places that should make this change as well.

From [autodoc_default_options](https://www.sphinx-doc.org/en/2.0/usage/extensions/autodoc.html#confval-autodoc_default_options) documentation:

> Setting None or True to the value is equivalent to giving only the option name to the directives.

`True` seemed more intuitive than `None`, so I replaced it this with:

```python
autodoc_default_options = {
    "members": True,
    "inherited-members": True,
    "show-inheritance": True,
}
```

The `sphinx-build` result appears to be the same as before.

Reply:
@hayesall Sorry, I probably was unclear, but we should keep **both** (deprecated `autodoc_default_flags` and new `autodoc_default_options`) config values to support all versions of Sphinx.

Reply:
I did misinterpret that. Both are included now 👍